build.gradle

```diff
@@ -4,6 +4,12 @@ plugins {
 apply plugin: 'java'
 apply plugin: 'scala'
 
+allprojects {
+    apply plugin: 'java'
+    sourceCompatibility = 1.6
+    targetCompatibility = 1.6
+}
```

Comment:
Do you think it is worthwhile just applying this to server/java-client so we can use java 8 in the it-tests?